das-dashboard/backend/services/container_services.py

```
@@ -161,15 +212,58 @@ def run_das_cli_command(self, command: list):
+                    "stdout": None,
+                    "stderr": result.stderr,
+                    "command": command,
+                }
+            raise DasCliCommandException(
+                f"Could not parse das-cli output as JSON: {output or '(empty)'}"
+            ) from e
+
+        except DasCliCommandException:
+            raise
 
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)